src/server/github_handler.rs

```diff
@@ -784,7 +785,8 @@ impl GithubEventHandler {
             }
 
             // Mark JIRAs as merged
-            if is_versioned_branch && !jira_projects.is_empty() {
+            let dest_jira_projects = self.config.repos().jira_projects(&self.data.repository, &branch_name);
```

Comment:
not sure the right name here

Reply:
`affiliated_jira_projects` ?

Reply:
yeah. though. we dont' even need to store them... will change to a bool.